packages/devextreme/scss/widgets/fluent/splitter/_colors.scss

```diff
@@ -1,7 +1,36 @@
 @use "sass:color";
 @use "../sizes" as *;
 @use "../colors" as *;
-
 // adduse
 
-$splitter-resize-handle-bg: #D1D1D1;
+$splitter-resize-handle-bg: #D1D1D1 !default;
+$splitter-resize-handle-color: black !default;
+@mixin state($property){
```

Comment:
Let's extract this mixin into base/splitter/_mixins.scss file and use it in all themes

Reply:
Correction: [commit](https://github.com/DevExpress/DevExtreme/pull/26938/commits/a66ce0f2e07b949b174d8f78795b1444e198b393)